keepercommander/commands/record_common.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""Code/Data shared between recordv3.py and record.py."""
+
+import urllib.parse
+
+import pudb
```

Comment:
Why this module was introduced into the application?

Reply:
As we slacked about, and as the docstring says, this new file has code shared by record.py and recordv3.py.  It has the advantages of avoiding code duplication, avoiding cyclic imports, and being thoroughly statically analyzed.

Reply:
Or do you mean urllib.parse?  Or pudb?  urllib.parse is there to extract data from a URL's query string.  pudb was a mistake; I've removed it.